frontend/eslint.config.js

```diff
@@ -20,6 +20,7 @@ export default [
     ],
     rules: {
       "no-control-regex": 0,
+      "@typescript-eslint/no-explicit-any": ["off"],
```

Comment:
I think the default config allows `Object` but not `any`

Reply:
This is because I don't wan't to explicitly handle every single individual error type